Author: Ole Streicher <debian@liska.ath.cx>
Description: Change license and copyright info for range.c
 The license written in the file is not DFSG compliant. I contacted the 
 author (Barry Brachman <brachman@dss.ca>)and got the permission to re-release
 the file under LGPL and accordingly change the copyright information.
--- a/install/printer/range.c
+++ b/install/printer/range.c
@@ -23,30 +23,25 @@
  * I've left the error checking in inrange; it is redundant (I hope!)
  * Nov/85 BJB
  * ========================================================================
+ * Copyright (c) 1985-2012 Distributed Systems Software
  *
- * Copyright (c) Barry Brachman
- *
- * Permission is given to freely copy or distribute this program
- * with the following restrictions:
- *
- *	1) The author is not responsible for the consequences of use of
- *		this software and is not responsible for correcting any defects
- *	2) The origin of this software must not be misrepresented, either by
- *		explicit claim or by omission
- *	3) You may not sell this program
- *	4) Altered versions must be plainly marked as such, and must not
- *		be misrepresented as being the original software
- *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  *
  * Barry Brachman
- * Dept. of Computer Science
- * Univ. of British Columbia
- * Vancouver, B.C. V6T 1W5
- *
- * .. {ihnp4!alberta, uw-beaver}!ubc-vision!ubc-cs!brachman
- * brachman@cs.ubc.cdn
- * brachman%ubc.csnet@csnet-relay.arpa
- * brachman@ubc.csnet
+ * Distributed Systems Software
+ * brachman@dss.ca
  * ========================================================================
  */
 /* $Id: range.c 16316 2003-07-16 03:38:11Z aips2adm $ */
--- a/install/printer/lwf_a.c
+++ b/install/printer/lwf_a.c
@@ -103,23 +103,25 @@
  * BJB - Nov/85
  * ===========================================================================
  *
- * Copyright (c) Barry Brachman
+ * Copyright (c) 1985-2012 Distributed Systems Software
  *
- * Permission is given to freely copy or distribute this program
- * with the following restrictions:
- *
- *      1) The author is not responsible for the consequences of use of
- *              this software and is not responsible for correcting any defects
- *      2) The origin of this software must not be misrepresented, either by
- *              explicit claim or by omission
- *      3) You may not sell this program
- *      4) Altered versions must be plainly marked as such, and must not
- *              be misrepresented as being the original software
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  *
  * Barry Brachman
- * Dept. of Computer Science
- * Univ. of British Columbia
- * Vancouver, B.C. V6T 1W5
+ * Distributed Systems Software
+ * brachman@dss.ca
  *
  * ===========================================================================
  *
